<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotAlignmentWidget</class>
 <widget class="QWidget" name="PlotAlignmentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="alignment_event_label">
     <property name="text">
      <string>Alignment Event:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="alignment_event_combo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="total_count_label">
     <property name="text">
      <string>Total: 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="window_size_label">
     <property name="text">
      <string>Window Size:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="window_size_spinbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.10000000000000001</double>
     </property>
     <property name="maximum">
      <double>999999.00000000000</double>
     </property>
     <property name="singleStep">
      <double>100.00000000000000</double>
     </property>
     <property name="value">
      <double>1000.0000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="interval_alignment_widget" native="true">
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="interval_alignment_layout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="interval_alignment_label">
        <property name="text">
         <string>Align to:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="interval_alignment_combo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Interval Beginning</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Interval End</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="offset_label">
     <property name="text">
      <string>Offset:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="offset_spinbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-999999.00000000000</double>
     </property>
     <property name="maximum">
      <double>999999.00000000000</double>
     </property>
     <property name="singleStep">
      <double>1.00000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
